mobile/examples/super_resolution/android/README.md

```diff
@@ -0,0 +1,43 @@
+# ONNX Runtime Mobile Super Resolution Android sample application with Ort-Extensions support for pre/post processing
+
+## Overview
+
+This is an basic Super Resolution example application for [ONNX Runtime](https://github.com/microsoft/onnxruntime) on Android with [Ort-Extensions](https://github.com/microsoft/onnxruntime-extensions) support for pre/post processing. The demo app accomplishes the task of recovering a high resolution (HR) image from its low resolution counterpart.
+
+The model used here is from source: [Pytorch Super Resolution](https://pytorch.org/tutorials/advanced/super_resolution_with_onnxruntime.html) and accomodated into [ONNX](https://github.com/onnx/onnx) version with pre/post processing support.
```

Comment:
briefly mentioned model usage here. 

will add more descriptions/pointers to tutorial when we write the general SuperRes README.md